tests/Dfe.PlanTech.Web.SeedTestData/ContentGenerators/SelfAssessmentPage.cs

```diff
@@ -0,0 +1,120 @@
+using Dfe.PlanTech.Domain.Content.Enums;
+using Dfe.PlanTech.Domain.Content.Models;
+using Dfe.PlanTech.Domain.Content.Models.Buttons;
+using Dfe.PlanTech.Domain.Questionnaire.Models;
+using Dfe.PlanTech.Infrastructure.Data;
+
+namespace Dfe.PlanTech.Web.SeedTestData.ContentGenerators;
```

Comment:
I _feel_ like we're breaking clean architecture here, or at least creating a dependency in the wrong direction by having seedTestData associated with our web project but without pulling it down and running it I could be wrong.

Does it feel 'right' to you?

Reply:
Aye, but we have seed data 'work', which is related only to testing, not under a test based namespace - I suppose what I'm getting at is what within this work _has_ a dependency which _requires_ it so sit where it does...?

Reply:
I went with this namespace as it relates to the web project, and it will work in tandem with something like 
PlanTech.Web.SeededE2ETests
which will be using this to get the database and then run Plan Tech Web against it for cypress testing, and that will depend on the web project so it seemed fitting that they go together

Though As Jim says it isn't a dependency so I'm good to rename it if theres something you think would be more suitable/ align more with the pattern - what would you recommend?

Reply:
No no it's fine if it makes sense and feels 'right'